docs/docs/references/configuration/cli/trivy_config.md

```diff
@@ -44,6 +44,7 @@ trivy config [flags] DIR
       --report string                     specify a compliance report format for the output (all,summary) (default "all")
       --reset-policy-bundle               remove policy bundle
   -s, --severity strings                  severities of security issues to be displayed (UNKNOWN,LOW,MEDIUM,HIGH,CRITICAL) (default [UNKNOWN,LOW,MEDIUM,HIGH,CRITICAL])
+      --show-suppressed                   [EXPERIMENTAL] show suppressed security findings
```

Comment:
I know you are planning to add support for misconfigurations in next PR. But let's hide this flag for `config` mode in this PR.

Reply:
Yes, I was just lazy 😄 Fixed.
https://github.com/aquasecurity/trivy/pull/6084/commits/32191f1f353b377efb1ebb3fcf5c2928a640633e